cmd/podman/shared/container.go

```diff
@@ -64,6 +94,201 @@ type ContainerSize struct {
 	RwSize     int64 `json:"rwSize"`
 }
 
+// NewBatchContainer runs a batch process under one lock to get container information and only
+// be called in PBatch
+func NewBatchContainer(ctr *libpod.Container, opts PsOptions) (PsContainerOutput, error) {
+	var (
```

Comment:
Why not just have a PsContainerOutput here instead?

Reply:
you mean define an empty one and use it through the func?